### PR TITLE
[FEATURE] Added highlight support for single pass

### PR DESCRIPTION
**Pull request by Deepak-Kesavan**
## What

Added highlight support for single pass

## Why

-

## How

-

## Can this PR break any existing features. If yes, please list possible items. If no, please explain why. (PS: Admins do not merge the PR without this section filled)

-

## Database Migrations

- 

## Env Config

- 

## Relevant Docs

-

## Related Issues or PRs

-

## Dependencies Versions

-

## Notes on Testing

-

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

File: backend/prompt_studio/prompt_studio_core_v2/prompt_studio_helper.py [hunk 1]
```diff
@@ -1157,7 +1157,11 @@ def _fetch_single_pass_response(
         tool_settings[TSPKeys.ENABLE_CHALLENGE] = tool.enable_challenge
         tool_settings[TSPKeys.ENABLE_HIGHLIGHT] = tool.enable_highlight
         tool_settings[TSPKeys.CHALLENGE_LLM] = challenge_llm
-
+        tool_settings[TSPKeys.ENABLE_HIGHLIGHT] = tool.enable_highlight
```
Comment (harini-venkataraman) [inline backend/prompt_studio/prompt_studio_core_v2/prompt_studio_helper.py:1160]: @Deepak-Kesavan Is this duplicated ? I see it is there in L1158

File: backend/prompt_studio/prompt_studio_core_v2/prompt_studio_helper.py [hunk 2]
```diff
@@ -1157,7 +1157,11 @@ def _fetch_single_pass_response(
...
+        tool_settings[TSPKeys.PLATFORM_POSTAMBLE] = getattr(
```
Comment (harini-venkataraman) [inline backend/prompt_studio/prompt_studio_core_v2/prompt_studio_helper.py:1161]: @Deepak-Kesavan Should these keys be added in prompt studio registry for export?

**Review by harini-venkataraman — APPROVED** [review]
Added few comments, LGTM otherwise

File: backend/prompt_studio/prompt_studio_core_v2/prompt_studio_helper.py (hunk 1 above)
Comment (Deepak-Kesavan) [reply backend/prompt_studio/prompt_studio_core_v2/prompt_studio_helper.py:1160]: Yes @harini-venkataraman . Removing it.

File: backend/prompt_studio/prompt_studio_core_v2/prompt_studio_helper.py (hunk 2 above)
Comment (Deepak-Kesavan) [reply backend/prompt_studio/prompt_studio_core_v2/prompt_studio_helper.py:1161]: These keys will be already there in tool export.

**Review by gaya3-zipstack — APPROVED** [review]
Looks good to me. 
PS: I have limited context on this one.

**Pull request merged**